.github/workflows/rebuilder.py

```diff
@@ -91,16 +144,19 @@ def get_pkgs(exclusions: set[str]) -> set[set]:
 
 def get_builds_from_copr(
     project_owner: str, project_name: str, copr_client: copr.v3.Client
-) -> list[dict]:
-    return copr_client.package_proxy.get_list(
-        project_owner,
-        project_name,
-        with_latest_succeeded_build=True,
-        with_latest_build=True,
-    )
+) -> list[CoprPkg]:
+    return [
+        CoprPkg(p)
+        for p in copr_client.package_proxy.get_list(
+            project_owner,
+            project_name,
+            with_latest_succeeded_build=True,
+            with_latest_build=True,
+        )
+    ]
```

Comment:
Does is make sense to make this a class method of `CoprPkg`? At least topic wise they both belong together. 

Also, the function is currently called `get_builds_from_copr` but it returns a list of Copr packages. I suggest to rename it `get_packages_from_copr`. 

So something like:

```
@classmethod
def get_packages_from_copr(cls, project_owner: str, project_name: str, copr_client: copr.v3.Client) -> list["CoprPkg"]:
    return [
        CoprPkg(p)       
        for p in copr_client.package_proxy.get_list(
            project_owner,
            project_name,
            with_latest_succeeded_build=True,
            with_latest_build=True,
         )
    ]
```